tutor/src/models/course/create.js

```diff
@@ -15,7 +15,7 @@ export default class CourseCreate extends BaseModel {
   @field num_sections = 1;
   @field estimated_student_count;
   @field is_preview = false;
-  @field is_college = false;
+  @field is_college = true;
   @field time_zone = 'Central Time (US & Canada)';
```

Comment:
This is a bug fix i snuck in - @Dantemss noticed we were setting all courses to not be college which is a despicable lie